app/controllers/ReportingController.scala

```diff
@@ -87,6 +95,59 @@ trait ReportingController extends BaseController {
     }
   }
 
+  private def makeRow(values: Option[String]*) =
+    values.map { s =>
+      val ret = s.getOrElse(" ").replace("\r", " ").replace("\n", " ").replace("\"", "'")
```

Comment:
I'm not sure there's a particularly more efficient way to do this - feel free to recommend something, i could combine \r and \n in a regex but i'm not sure it's a substantial gain